### ISSUE TITLE
Openlayers Tutorial

### ISSUE THREAD

**Issue by MatthaeusMarte** [tuwien-geoweb-2015/ifip-block5#1]
Hallo Herr Hocevar, ich würde gerne das tutorail von openlayers machen damit ich halbwegs verstehe wie man eine karte macht.... mein git node macht mich einfach fertig :/ es funktioniert einfach nicht... ich kann nicht mal das tutorial starten... ich hoffe sie können mir weiter helfen lg
![image](https://cloud.githubusercontent.com/assets/15378013/11190813/13f63564-8c98-11e5-8b71-eaffb4b4645e.png)

**Comment by ahocevar** [comment]
Ja, es ist tatsächlich frustrierend, dass dieses Node-Zeugs auf Windows so unzuverlässig ist. Ich stelle euch [hier ein zip](https://dl.dropboxusercontent.com/u/5489321/openlayers-workshop.zip) mit einem openlayers-workshop Verzeichnis zur Verfügung, wo `npm install` schon erfolgreich ausgeführt würde. Damit sollte dann eigentlich `npm start` problemlos funktionieren. Falls nicht, bitte wieder hier melden.

**Comment by MatthaeusMarte** [comment]
danke für die rasche meldung scheint bis jetzt zu funktionieren... lisa Rei
hat ihnen auch noch eine frage gschrieben..
https://github.com/tuwien-geoweb-2015/ifip-block5/issues/2

lg

m marte

Am 16. November 2015 um 20:21 schrieb Andreas Hocevar <
notifications@github.com>:
